src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/RazorComponentsWeb-CSharp.App.csproj.in

```diff
@@ -10,7 +10,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Components.Browser" Version="${MicrosoftAspNetCoreComponentsPackageVersion}" />
     <PackageReference Include="Microsoft.AspNetCore.Components.Build" Version="${MicrosoftAspNetCoreComponentsPackageVersion}" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.NET.Sdk.Razor" Version="${MicrosoftNETSdkRazorPackageVersion}" PrivateAssets="All" />
```

Comment:
If this is being removed from here, can it also be removed from all the other `.csproj` files under `src/Components`, including the Blazor templates in `src\Components\blazor\src\Microsoft.AspNetCore.Blazor.Templates\content`?

Reply:
We usually leave the reference around in test apps. The template tests insert a reference to the package when running: https://github.com/aspnet/AspNetCore/blob/master/src/Templating/test/Templates.Test/Helpers/TemplateTestBase.cs#L113-L127 to simulate what we do in our checked in code.

Reply:
Will do.